scripts/sync-openapi.ts

```diff
@@ -0,0 +1,69 @@
+import fs from "fs/promises";
+
+const SOURCE_URL =
```

Comment:
This should _probably_ be defined via an env var, so we can swap it out if we end up restructuring the repo.